### PR TITLE
Error disc

### PR DESCRIPTION
**Pull request by makeclean**
Discussion of errors

### CODE REVIEW

File: src/report.tex [hunk 1]
```diff
@@ -211,28 +211,60 @@ \section*{Executive Summary}
...
+to a more extensive program of dose reduction in the interspaces. 
 \\
 \\
 This analysis focused on only shutdown photon sources, and therefore does not
-contain prompt photon doses which are not present following machine shutdown.
+contain prompt photon doses which are not present following machine shutdown. It
+does not contain neutrons that are the product of decay reactions, such as neutron
+production from activated water, which is a small component compared to the plasma
+source. The uncertainty in the \gls{sdr} was estimated being between a relative 
+error of between 0.5 and 2.0.
```
Comment (gonuke) [inline src/report.tex]: this mixes uncertainty and relative error....  How about:
"Although it is not possible to rigorously asses the uncertainty in a calculation of this scope, the uncertainty has been estimated as a factor of 2.  That is, the actual results are expected to be between 0.5 times and 2 times the simulated results."

File: src/report.tex [hunk 2]
```diff
@@ -211,28 +211,60 @@ \section*{Executive Summary}
...
 \\
 \\
-Future studies should focus on further decreasing the \gls{sdr} in the person 
+Future analysis should focus around improving analysis 1) Methods to decrease
```
Comment (gonuke) [inline src/report.tex]: probably no capitalization here

Comment (gonuke) [inline src/report.tex]: "Future analysis should focus on improvements such as: "

File: src/report.tex [hunk 3]
```diff
@@ -211,28 +211,60 @@ \section*{Executive Summary}
...
 \gls{sdr} comes from decay photons created outside the upper port, from the activation
 of the structural components nearby. Focus should be placed on improving 
-shielding internal to the strong sources, possible on improving the \gls{vv} filler
-near the \gls{pfc}2 or strong local shielding. 
+shielding internal to the strong sources, possible on improving the \gls{vv} 
+filler near the \gls{pfc}2 or strong local shielding. Further B$_4$C additions 
+should be considered around \gls{pfc}2 and potentially in the regions between 
+the vessel extensions.  
+\\
+\\
```
Comment (gonuke) [inline src/report.tex]: drop everything below this in this section

File: src/report.tex [hunk 4]
```diff
@@ -2191,7 +2223,33 @@ \subsection{Estimation of Uncertainties}
...
+estimate below.
+
+1 - Neutronics Benchmarks - Several neutronics benchmarks have been performed 
+\cite{Kodeli2004717,Batistoni19941724,Fausser20112135,Batistoni1995504,Batistoni1997233,Flammini20151964,Klix2016,Kodeli2016,Angelone2016}
+and the majority show deviations from the experimental result of at between 5-50\%, we therefore 
+can ascribe a lower bound uncertainty of 5\% and an upper reasonable bound of 20\% in the neutron 
+result purely from nuclear data.
+2 - Activation Data Uncertainty - Activation data has uncertainty in the reaction channel encapsulated 
+as part of it, often as part of activation calculations the inventory code estimate in the uncertainty 
+in the nuclide due to all the formation channels, this is in the range 10-100\% depending upon the nuclide,
```
Comment (gonuke) [inline src/report.tex]: this sentence gets lost in itself

File: src/report.tex (hunk 3 above)
Comment (makeclean) [inline src/report.tex]: Is that everything from line 236 to 260? Or From lines 236 to 239?

File: src/report.tex [hunk 5]
```diff
@@ -2191,7 +2223,33 @@ \subsection{Estimation of Uncertainties}
...
+3 - Mesh Resolution - There has been only one study showing the effect of the activation mesh upon the final 
+result in an SDR calculation \cite{r2smesh}, and for cell based methods \cite{frej}, these suggest that the mesh may have 
+an uncertainty of 10\% at its lowest and an upper bound of 40\%
+4 - Neutron Histories - A target uncertainty in the total neutron flux for this calculation was 
+10\%, however this does not ascribe any uncertainty limits to the neutron spectrum, a reasonable 
+estimate of the effect is difficult to summarize, but a relative error of 10\% directly form the 
+flux as a lower bound and an upper bound of 20\% is ascribed.
+\\
+\\
+The smallest the total relative error could be therefore 0.45 and the highest reasonable estimate is 2.0
```
Comment (gonuke) [inline src/report.tex]: Are we mixing factors here....
I think we could have exactly the right answer.... (unlikely)
A generous estimate of error suggests that we are off by a factor of 1.45 (the right answer is between 1/1.45 and 1.45 x the estimate)
A best estimate of error suggests that we are off by a factor fo 2 (the right answer is between 1/2 and 2x the estimate)

File: src/report.tex (hunk 3 above)
Comment (gonuke) [inline src/report.tex]: through 260

File: src/report.tex (hunk 4 above)
Comment (gonuke) [inline src/report.tex]: I think the punctuation needs some work

File: src/report.tex (hunk 5 above)
Comment (makeclean) [inline src/report.tex]: Quite right

File: src/report.tex [hunk 6]
```diff
@@ -2191,7 +2199,33 @@ \subsection{Estimation of Uncertainties}
...
-of institutions world wide.
+of institutions world wide. 
+\\
+\\
+We can however attempt to estimate and lower bound and a reasonable upper bound. Estimation of the uncertainty 
+in the SDR is challenging due to a number of sources of uncertainty which may compensate each other in unexpected 
+ways. The correct method to determine the uncertainty in the SDR has not been performed, however we attempt an
+estimate below.
+
+1 - Neutronics Benchmarks - Several neutronics benchmarks have been performed
```
Comment (gonuke) [inline src/report.tex]: should this be an {enumerate}

Comment (gonuke) [inline src/report.tex]: sorry to drag it out...

File: src/report.tex [hunk 7]
```diff
@@ -2191,7 +2199,33 @@ \subsection{Estimation of Uncertainties}
...
+3 - Mesh Resolution - There has been only one study showing the effect of the activation mesh upon the final 
+result in an SDR calculation \cite{r2smesh}, and for cell based methods \cite{frej}, these suggest that the mesh may have 
+an uncertainty of 10\% at its lowest and an upper bound of 40\%
+4 - Neutron Histories - A target uncertainty in the total neutron flux for this calculation was 
+10\%, however this does not ascribe any uncertainty limits to the neutron spectrum, a reasonable 
+estimate of the effect is difficult to summarize, but a relative error of 10\% directly form the 
+flux as a lower bound and an upper bound of 20\% is ascribed.
+\\
+\\
+The smallest the total relative error could be therefore 1.45 and the highest reasonable estimate is 2.0
```
Comment (gonuke) [inline src/report.tex]: Still worried about the interpretation of these numbers....
Should we give the examples of 1/1.45 to 1.45 and 1/2 to 2?

Comment (gonuke) [inline src/report.tex]: these numbers results in asymmetric uncertainties - so that's the problem

**Comment by gonuke** [comment]
Go! Thanks for hanging in there... let's send this off to Brad and Mike and Ahmad as final

**Pull request merged**